pkgs/applications/networking/cluster/tgswitch/default.nix

```diff
@@ -1,5 +1,5 @@
-{ buildGoPackage, lib, fetchFromGitHub }:
-buildGoPackage rec {
+{ buildGoModule, lib, fetchFromGitHub }:
+buildGoModule rec {
```

Comment:
Haven't been following the buildGoModule vs buildGoPackage conversation. Is the recommended practice now to use buildGoModule ?

Reply:
It's marked as legacy now.

Per nixpkgs manual:
> The function buildGoPackage builds legacy Go programs, not supporting Go modules.

Reply:
Thanks!